configuration/OVERVIEW.md

```diff
@@ -584,6 +585,83 @@ return result;
 Note that you can pass a large object to the form, which can then read any value, but doing so does noticeably slow the loading of the form. Because of this it is preferable to remove from the context any fields that are not being used. It is a good idea to future proof by maintaining the same structure so that fields can be added without needing to modify existing form calculations.
 ### Tips & Tricks <!-- TODO -->
 ### Troubleshooting <!-- TODO -->
+### Create A New Collect App
+
+#### Step 1: Download Android studio or Eclipse. 
+http://askubuntu.com/questions/634082/how-to-install-android-studio-on-ubuntu
+
+
+Once installed, start Android Studio on Windows, from the Unity Dash(Ubuntu) or just run command
+
+*N:B Run below command as `sudo`*
+```sudo /opt/android-studio/bin/studio.sh```
+![alt-text](img/android-studio-splash-screen.png "Android Studio Splash Screen")
+
+#### Step 2: Clone Medic-Collect and open the folder as an existing Android Studio Project
+By this time hopefully you have already cloned the `medic-collect` repo. If not, run the command `git clone https://github.com/medic/medic-collect.git`
+
+On Android Studio, we are going to open the `medic-collect` project by navigating to it and clicking on the root folder which will be reprensented on the Android Studio file navigator by an Android Studio Project Icon and then clicking Ok.
+
+![alt-text](img/Android-Studio-File-Navigator.png "Open Medic-Collect as Android Project")
+
+Let Gradle build and load project dependencies.
+![alt-text](img/Load-Project-Dependencies.png "Load Project Dependencies")
+
+#### Step 3: Make sure that the folder to your medic collect flavor has the forms and configuration necessary for medic-collect to build the apk.
+
+Do this by navigating to `medic-workspace/medic-projects/_medic-collect-flavors/`and creating the folder `le_app`.
+Copy paste the contents of another collect flavor. Copy from `standard`. This is so that you can maintain the proper folder structure needed. We will replace these contents customise it to `le-app`.

```

Comment:
It could be better to explicitly indicate the ideal folder hierarchy. An illustration would help